### DISCUSSION TITLE
The Icon file converted from the SVG file does not render properly in the Application Icon of the Form.

### DISCUSSION THREAD

**Discussion by BagavathiPerumalA** [vvvv/SVG#1135] [Q&A]
#### Description

The Icon file converted from the SVG file does not render properly in the Application Icon of the Form. Need to convert the SVG file to Icon by using SvgDocument. But, unable to render the icon file as the Form application icon. How to resolve this issue.

#### Example data

Converted-Icon-From-SVG:

![Converted-Icon-From-SVG](https://github.com/svg-net/SVG/assets/93652794/cbc8a98a-34ae-40c3-bfc2-fa0fb6021497)

```
string svgFilePath = @"Images\SVG-Cloud.svg";

string outputIcoPath = @"Images\SVG-Cloud.ico";

int iconWidth = 32;
int iconHeight = 32;
try
{
    SvgDocument svgDocument = SvgDocument.Open(svgFilePath);

    if (svgDocument != null)
    {
        Bitmap bitmap = svgDocument.Draw(iconWidth, iconHeight);

        if (bitmap != null)
        {
            bitmap.Save(outputIcoPath, System.Drawing.Imaging.ImageFormat.Icon);

            Icon icon = Icon.ExtractAssociatedIcon(outputIcoPath);

            this.Icon = icon;
        }
        else
        {
            MessageBox.Show("Error: Bitmap could not be created from SVG.", "Error", MessageBoxButtons.OK, MessageBoxIcon.Error);
        }
    }
    else
    {
        MessageBox.Show("Error: SVG document could not be loaded.", "Error", MessageBoxButtons.OK, MessageBoxIcon.Error);
    }
}
catch (Exception ex)
{
    MessageBox.Show($"Error: {ex.Message}", "Error", MessageBoxButtons.OK, MessageBoxIcon.Error);
}

```

#### Used Versions

**Operating system:** Windows 11

**SVG NuGet version:** 3.4.6

**.Net version:** 4.7.2

Sample: 
[SVG-ICON.zip](https://github.com/svg-net/SVG/files/14039639/SVG-ICON.zip)

**Comment by paulushub** [comment]
Use this https://github.com/HerbertLausmann/HL.IconPro,  it is based on SVG-NET.

**Comment by RaghavendraSudhakar** [comment]
@paulushub A file named HL.MVVM is missing in this GitHub file https://github.com/HerbertLausmann/HL.IconPro

![image](https://github.com/svg-net/SVG/assets/153831960/95a2b0e4-97e6-4e9e-bcb6-9e5a1fcccb3f)

**Reply by paulushub** [reply]
https://github.com/HerbertLausmann/HL.MVVM
